bukkit/src/main/resources/config.yml

```diff
@@ -558,6 +558,9 @@ auto-op: false
 # - Set to false to only allow users who have the permissions access to the commands
 commands-allow-op: true
 
+#Stops Luckperms Commands from being run from non player sources. 
+#eg: Stops a mod from running /lp user Example permission set *
+block-non-player-sources: false
```

Comment:
Again - please match the style found in the rest of the file.

`# Stops LuckPerms commands from being....` - space after the `#`, proper capitalisation.....

Should also be a new-line between the setting and the start of the next comment.